google/cloud/functions/framework.h

```diff
@@ -54,12 +51,7 @@ using HttpFunction =
  */
 int Run(int argc, char const* const argv[], HttpFunction handler) noexcept;
```

Comment:
The comment on this function still references the `functions_internal` namespace. Is it still intended that the application does not use this directly? If so, why move this out of the `functions_internal` namespace?

Reply:
I am going to revert that part of this change in a minute, but we should talk more elsewhere.

Admittedly, I am not super certain about this, but I suspect it depends on what we mean by "the application",  I think we have two separate use cases:

1. Cloud Functions applications never use this directly, we build their function and then pass their function to this `Run()` thing.
1. We want (I think) to release this library as a standalone thing, at least we envisioned this library being released as a standalone thing.  Why?  To support:
    - Local testing and development: Cloud Functions developers may need to test their code before deploying to production.
    - Support users that start with Cloud Functions (and write everything to it), but for whatever reason their application grows too complex for it (maybe they gained a binary-only dependency, or they need more threads, or something).  In that case they may want to keep using the Cloud Functions framework, but build using their build scripts and deploy to Cloud Run.